java/src/main/java/com/powsybl/dataframe/network/NetworkDataframes.java

```diff
@@ -87,55 +87,55 @@ private static Map<DataframeElementType, NetworkDataframeMapper> createMappers()
         return Collections.unmodifiableMap(mappers);
     }
 
-    static <U extends Injection<U>> ToDoubleBiFunction<U, NetworkDataframeContext> getPerUnitP() {
+    static <U extends Injection<?>> ToDoubleBiFunction<U, NetworkDataframeContext> getPerUnitP() {
```

Comment:
Why is it necessary to change this ?

Reply:
Because otherwise signature is not compatibles with what is expected in doubles() function when building the dataframe